docs/src/users_guide/QuantumObject/QuantumObject.md

```diff
@@ -39,9 +39,12 @@ Qobj(rand(4, 4))
 ```
 
 ```@example Qobj
-Qobj(rand(4, 4), dims = [2, 2])
+Qobj(rand(4, 4), dims = (2, 2))
 ```
```

Comment:
maybe write this example block as:
```
Qobj(rand(ComplexF64, 4, 4), dims = [2, 2])  # dims as Vector
Qobj(rand(ComplexF64, 4, 4), dims = (2, 2))  # dims as Tuple
Qobj(rand(ComplexF64, 4, 4), dims = SVector(2, 2)) # dims as StaticArrays.SVector
```
I'm thinking of reexporting `SVector`, so that the users can directly call `SVector`.
I also add `ComplexF64` here to show the people from python that we can specify element types in julia.

Reply:
Actually I don't know if it is useful to export `SVector`. I don't think that it would be used, one can always use a `Tuple` normally.

I think that people who need `SVector` they need it for other reasons, like dynamics of very small systems where `SVector` is useful.

Even if re-exported, it is much simple to write `Qobj(A, dims=(2,2))` instead of `Qobj(A, dims=SVector(2,2))`. Do you agree?

Reply:
as we discussed in Zulip, we can resolve this